### DISCUSSION TITLE
Not Found - STOP elementFinder

### DISCUSSION THREAD

**Discussion by Wielebny666** [Dhruv-Techapps/acf-docs#317] [General]
I made a simple automation that will click the hCaptcha checkbox.
Find the item #checkbox and click on it.

Testing it at https://accounts.hcaptcha.com/demo

`{
  "actions": [
    {
      "addon": {
        "condition": "",
        "elementFinder": "",
        "recheck": 0,
        "recheckInterval": 0,
        "recheckOption": "stop",
        "value": "",
        "valueExtractor": ""
      },
      "elementFinder": "#checkbox",
      "elementType": "",
      "enable": true,
      "focus": true,
      "initWait": 0,
      "name": "",
      "repeat": 0,
      "repeatInterval": 0,
      "settings": {
        "iframeFirst": true
      },
      "value": "MouseEvents::click"
    }
  ],
  "batch": {
    "refresh": false,
    "repeat": 0,
    "repeatInterval": 5
  },
  "enable": true,
  "hotkey": "Ctrl + Shift + A",
  "initWait": 0,
  "loadType": "window",
  "name": "hcaptcha",
  "spreadsheetId": "",
  "startTime": "",
  "startType": "auto",
  "url": "https://accounts.hcaptcha.com/demo"
}`

I get an error in the console:
Not Found - STOP elementFinder: #checkbox

Am I doing something wrong?

regards

**Comment by dharmesh-hemaram** [comment]
Change the url to https://newassets.hcaptcha.com/captcha/ aa its within an iframe . check out #21 #33 #198
